ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BlockAttestationsPool.java

```diff
@@ -39,18 +40,16 @@
   //  private final ConcurrentHashMap<Bytes32, UnsignedLong> dataRootToSlot = new
   // ConcurrentHashMap<>();
 
-  private final ConcurrentHashMap<Bytes32, Bitlist> unprocessedAttestationsBitlist =
+  final ConcurrentHashMap<Bytes32, Bitlist> unprocessedAttestationsBitlist =
       new ConcurrentHashMap<>();
 
-  private final ConcurrentHashMap<Bytes32, Bitlist> processedAttestationsBitlist =
+  final ConcurrentHashMap<Bytes32, Bitlist> processedAttestationsBitlist =
       new ConcurrentHashMap<>();
 
   private final int QUEUE_INITIAL_CAPACITY =
-      Constants.MAX_VALIDATORS_PER_COMMITTEE
-          * Constants.MAX_COMMITTEES_PER_SLOT
-          * Constants.SLOTS_PER_EPOCH;
+      MAX_VALIDATORS_PER_COMMITTEE * Constants.MAX_COMMITTEES_PER_SLOT * Constants.SLOTS_PER_EPOCH;
```

Comment:
We should probably just set the initial capacity to something "reasonable" (1024?) - we won't necessarily fill up the queue to its max size.

Reply:
done